explore/src/main/scala/explore/config/TableMaker2.scala

```diff
@@ -0,0 +1,263 @@
+package reactST.reactTable
```

Comment:
I split TableMaker in 2: a base trait with generic logic and an impementation that renders HTML tables. Also, there's the SUITableMaker now which is an implementation that renders SUI tables.

I'll be sending a PR to `scalajs-react-table` soon.